macros/string_index_based_replacer.go

```diff
@@ -102,3 +102,21 @@ func (s *stringIndexBasedReplacer) getTemplate(url string) urlMetaTemplate {
 	}
 	return template
 }
+
+func (s *stringIndexBasedReplacer) ReplaceBytes(result *strings.Builder, url string, macroProvider *MacroProvider) {
+	tmplt := s.getTemplate(url)
```

Comment:
Nitpick: can this variable name just be `template`

Reply:
Done